### DISCUSSION TITLE
GitHub actions redacts repository owner when it's not a secret

### DISCUSSION THREAD

**Discussion by mimartin12** [community/community#121135] [Actions]
### Select Topic Area

General

### Body

I am attempting to link a commit in the GitHub summary of a workflow run, but the workflow continues to mask the value of the repository owner, as if it was a secret. How can I get this value into plain text and why does GitHub actions keep masking this? 

```yaml
      - name: Display commit SHA
        run: |
          echo \
          ":steam_locomotive: View [commit](${{ github.server_url }}/${{ github.repository }}/commit/${{ needs.artifact-check.outputs.artifact-build-commit }})" \
          >> $GITHUB_STEP_SUMMARY
```

Results in this URL: [commit](https://github.com/***/clients/commit/dba910d0b946176344f13f55e081b3a7965ee9f4)

**Comment by grandmas-favorite** [comment]
Hi @mimartin12!

It's likely [a permissions issue](https://docs.github.com/en/get-started/learning-about-github/access-permissions-on-github) - are you the repo owner, contributor, or have no access?
